### DISCUSSION TITLE
Increase Font size in Python

### DISCUSSION THREAD

**Discussion by harishshivaraj** [perspective-dev/perspective#2271] [General]
Hey all

I'm using Perspective in some of my projects, I just cant figure out how to use the plugin view config correctly. That is, what are the available configuration for the users? 

So for instance I would like to increase the font size of the table. How can I do that? 

Many thanks

**Comment by timkpaine** [comment]
font size is controlled by the browser and the css assets on your page, completely separate from python. 

The best way to figure out how to configure the viewer is to play around with the viewer in the UI (I like to use JupyterLab) then look at the value of `widget.save()` which contains the serialized viewer configuration you just manually manipulated ( in javascript you can do `the_dom_node_for_perspective_viewer.save().then((res) => console.log(JSON.stringify(res)))` )

**Comment by texodus** [comment]
Font size is not currently a user configurable style, it is a theme-configurable style though, e.g. it can be accomplished via CSS, see the [NFT  example](https://github.com/sc1f/pudgy-penguin-perspective/blob/master/src/index.css#L95).

Your browser's accessibility controls can be used to scale Perspective's interface consistently if you just need accessibility..

**Comment by harishshivaraj** [comment]
Cheers guys. I got this resolved now. Just had go with the custome css.

**Comment by harishshivaraj** [comment]
closed
